src/components/ProjectDialog.tsx

```diff
@@ -80,13 +82,13 @@ const ProjectDialog: React.FC<ProjectDialogProps> = ({
               </Carousel>
              <Grid container spacing={3} sx={{marginBottom: '32px', marginTop: '32px'}}>
                    <Grid item xs={12} md={6}> 
-                      <Typography
-                      variant='body1'
-                      sx={{color: '#445E77', letterSpacing: '-0.5px', marginBottom: '24px'}}
-                      >
-                      <ReactMarkdown
-                      >{project.description}</ReactMarkdown>  
-                      </Typography>
+                      <Box
+                          sx={{color: '#445E77', letterSpacing: '-0.5px', marginBottom: '24px', fontFamily: "'Figtree Variable',sans-serif"}}
+                          >
+                      <ReactMarkdown>
```

Comment:
should this have `className='markdown-wrapper'` too?

Reply:
No. markdown-wrapper actually clamps lines - i thought it affected link and code styling. Rename